.github/workflows/rerun_workflow_on_push.yml

```diff
@@ -27,13 +28,12 @@ jobs:
         shell: bash
         run: |
           base_branch="${GITHUB_REF#refs/heads/}"
-          if [ $base_branch = main ]; then
+          if [[ $base_branch == "main" ]] || [[ $base_branch == "feature/"* ]]; then
```

Comment:
I think `"feature/"*` should be `"feature/*"` here, right?

Reply:
i dont think. [Example](https://stackoverflow.com/a/19897118/19839167)